### PR TITLE
Enable Package support for RHEL OSP

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,26 @@
...
+  tags: selinux
+
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+  when: ansible_distribution == "RedHat"
+  tags: rhn
+
```
Comment (nirajdp76) [inline roles/common/tasks/main.yml]: Need to attach to a pool before we can enable the repos

- name: Add OSP pool
  command: subscription-manager attach --pool={{ item }}
  when: ansible_distribution == "RedHat"
  with_items:
  - 8a85f98158880f8d01588bde0f7c3c43

Comment (paulczar) [inline roles/common/tasks/main.yml]: if you look at the following task it runs attach  ( don't need to specify the pool ).

https://github.com/blueboxgroup/ursula/pull/2383/files/4c234a0ea932cfca7dfcc9979b01f48b08f0c56f#diff-3cd6f6e3b6e4c2ea1695765d4b707c63R39

Comment (nirajdp76) [inline roles/common/tasks/main.yml]: odd, not sure how this is working without specifying which pool to attach to. Anyways, @twaldrop PR on paul_fixes replaces this code.

File: envs/example/defaults-2.0.yml [hunk 2]
```diff
@@ -1,11 +1,10 @@
 ---
 stack_env: example
 ursula_env_path: "{{ lookup('env','URSULA_ENV') }}"
-
 country_code: US
 
 openstack_install_method: 'source'
-#openstack_install_method: 'package'
+basevenv_lib_dir: "/opt/openstack/base/lib/python0.7/site-packages"
```
Comment (nirajdp76) [inline envs/example/defaults-2.0.yml]: There is typo in the path.. should be python2.7

File: roles/common/tasks/python.yml [hunk 3]
```diff
@@ -3,91 +3,125 @@
...
-- name: pip install python-dateutil
-  pip: name=python-dateutil
-  register: result
-  until: result|succeeded
-  retries: 5
+- block:
+  - name: install additional python packages
+    git: repo={{ item.repo }} dest=/opt/stack/{{ item.name }}
+         version={{ item.rev }} accept_hostkey=True
+    with_items: "{{ common.python_extra_packages }}"
```
Comment (chengtcli) [inline roles/common/tasks/python.yml]: seems common.python_extra_packages doesn't have property of `repo` and `name`

**Pull request merged**